src/Symfony/Component/Form/Extension/Core/DataTransformer/DateIntervalToArrayTransformer.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+
+/**
+ * Transforms between a normalized date interval and an interval string/array.
+ *
+ * @author Steffen Roßkamp <steffen.rosskamp@gimmickmedia.de>
+ */
+class DateIntervalToArrayTransformer implements DataTransformerInterface
+{
+    private $fields;
+    private $availableFields = array(
```

Comment:
This can be made static as it actually never changes.

Same done in https://github.com/MisatoTremor/symfony/blob/add_dateinterval_type_2_8/src/Symfony/Component/Form/Extension/Core/DataTransformer/PercentToLocalizedStringTransformer.php#L29